server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1516,6 +1516,114 @@
         </complexContent>
     </complexType>
 
+    <complexType name="httpProxyServicePropertiesType">
+        <complexContent>
+            <restriction base="gateway:ServicePropertiesType">
+                <sequence>
```

Comment:
Follow up from Robin, allow any ordering.  Any validation we can do on top of that is bonus